compose/docker-compose.base.yml

```
@@ -11,8 +11,34 @@ networks:
+
+  redis-data:
+    driver: local
+    name: redis-data
+
+  redis-insight:
+    driver: local
+    name: redis-insight
+
+  smtp:
+    driver: local
+    name: smtp-dev
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)